dhis-2/dhis-api/src/main/java/org/hisp/dhis/period/PeriodType.java

```diff
@@ -646,6 +646,17 @@ public Period createPeriod( String isoDate )
     // CalendarPeriodType
     // -------------------------------------------------------------------------
 
+    public final Period getFuturePeriod( int periodOffset )
```

Comment:
I'm not sure what is gained by moving this logic from `DataElement`. It still has the funny concept of "Future" which is different from "Next" by -1. If moved, this public method needs JavaDoc to explain what it does.

Reply:
I needed to move this since I call this from validation were I don't use the data element's period type.

Reply:
done